change/@ni-nimble-components-54d4e9df-8176-49b5-9021-269b73561000.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is there a new (or newly discovered) expectation on how clients set the table's height? If so should we try to document it in the table's story?

Something like:
## Sizing
Clients must provide an explicit `height` or `min-height` for `nimble-table`. Do not leave both values set to `auto`.